.circleci/config.yml

```diff
@@ -180,91 +147,187 @@ jobs:
             export TRAVIS_BUILD_NUMBER="${CIRCLE_BUILD_NUM}"
             export TRAVIS_COMMIT="${CIRCLE_SHA1}"
             sbt docs/docusaurusPublishGhpages
-      - save_cache:
-          key: sbt-cache
-          paths:
-            - "~/.ivy2/cache"
-            - "~/.sbt"
-            - "~/.m2"
-            - "~/website/node_modules"
+      - <<: *save_cache
+
+jobs:
+  lint:
+    <<: *lint
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_212
+      - <<: *jdk_8
+
+  mdoc:
+    <<: *mdoc
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_212
+      - <<: *jdk_8
+
+  test_211_jdk8_jvm:
+    <<: *testJVM
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_211
+      - <<: *jdk_8
+
+  test_212_jdk8_jvm:
+    <<: *testJVM
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_212
+      - <<: *jdk_8
+
+  test_dotty_jdk8_jvm:
+    <<: *testJVM
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_dotty
+      - <<: *jdk_8
+
+  test_211_jdk11_jvm:
+    <<: *testJVM
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_211
+      - <<: *jdk_11
+
+  test_212_jdk11_jvm:
+    <<: *testJVM
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_212
+      - <<: *jdk_11
+
+  test_dotty_jdk11_jvm:
+    <<: *testJVM
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_dotty
+      - <<: *jdk_11
+
+  test_211_jdk8_js:
+    <<: *testJS
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_211
+      - <<: *jdk_8
+
+  test_212_jdk8_js:
+    <<: *testJS
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_212
+      - <<: *jdk_8
+
+  test_212_jdk11_js:
+    <<: *testJS
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_212
+      - <<: *jdk_11
+
+  test_211_jdk11_js:
+    <<: *testJS
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_211
+      - <<: *jdk_11
+
+  release_211:
+    <<: *release
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_211
+      - <<: *jdk_8
+
+  release_212:
+    <<: *release
+    <<: *machine_ubuntu
+    environment:
+      - <<: *scala_212
+      - <<: *jdk_8
+
+  documentation:
+    <<: *documentation
+    <<: *machine_ubuntu
+    environment:
+      - <<: *jdk_8
+
+  microsite:
+    <<: *microsite
+    <<: *machine_ubuntu
+    environment:
+      - <<: *jdk_8
 
 workflows:
   version: 2
   build:
     jobs:
-      - lint212_jdk8:
-          filters:
-            tags:
-              only: /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
```

Comment:
@NeQuissimus any reason to remove this? By default, builds are not triggered on tags : https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag
I just tried to cut a release and nothing happened indeed.

Reply:
Oh crap, I had meant to extract the regex' into a reference but forgot to out it back 😔